__tests__/app.test.js

```diff
@@ -169,6 +164,63 @@ test("Testing for a 404 error with a invalid review_id that does not exist", ()
     });
 });
 
+test("GET /api/reviews?query accepts a sort_by query, which sorts by title", () => {
+  return request(app)
+    .get("/api/reviews?sort_by=title")
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("title", { descending: true });
+    });
+});
+test("accepts query and returns the reviews in default order (date and descending)", () => {
+  return request(app)
+    .get(`/api/reviews`)
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("created_at", { descending: true });
+    });
+});
+test("accept a category and return games in that category", () => {
+  return request(app)
+    .get(`/api/reviews?category=social+deduction`)
+    .expect(200)
+    .then(({ body: { reviews } }) => {
+      expect(reviews).toBeSortedBy("title", { descending: false });
+    });
+});
```

Comment:
if you're checking for the category - you should check that all of the returned reviews definitely have that category property.

Reply:
Had an absolute mare with this, spotted it when doing the console.logs to add examples to my json! Sorted now :D